### PR TITLE
Bind to observers

### PR DESCRIPTION
**Pull request by intmain**
bind 한번에 여러개 하도록 추가해주세요

### CODE REVIEW

**Comment by tokijh** [comment]
Could you please add some unit tests on this class?

File: RxSwiftExtensions/RxCocoa/RxCocoa_Binder.swift [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+//  Copyright © 2018 tokijh. All rights reserved.
+//
+
+import RxSwift
+import RxCocoa
+
+extension ObservableType {
+    public func bind<O: ObserverType>(to observers: [O]) -> Disposable where O.E == E {
+        let shared = self.share()
+        let disposables = observers.map({ shared.bind(to: $0) })
```
Comment (cruisediary) [inline RxSwiftExtensions/RxCocoa/RxCocoa_Binder.swift]: How about this? 👻 
```swift
let disposables = observers.map { shared.bind(to: $0) }
```

Comment (tokijh) [reply RxSwiftExtensions/RxCocoa/RxCocoa_Binder.swift]: Not bad. But I prefer to use `({ })` for things that can be expressed in one line.
More than two lines, use `{ }` What do you think?

Comment (cruisediary) [reply RxSwiftExtensions/RxCocoa/RxCocoa_Binder.swift]: @tokijh It seems to be possible to express in one line

**Comment by intmain** [comment]
그냥 해주세요.

File: RxSwiftExtensions/RxCocoa/RxCocoa_Binder.swift (hunk 1 above)
Comment (tokijh) [reply RxSwiftExtensions/RxCocoa/RxCocoa_Binder.swift]: @cruisediary I don't care. Please make a PR If you want to fix it 👍

**Pull request merged**